lib/app/services/media_service/media_service.c.dart

```diff
@@ -44,6 +45,74 @@ class MediaFile with _$MediaFile {
 typedef CropImageUiSettings = List<PlatformUiSettings>;
 
 class MediaService {
+  final Map<String, AssetPathEntity> _albumsCache = {};
```

Comment:
wdyt about creating another service for albums, because media service is looks like directly related with raw media types.

Reply:
make sense, done